hudi-common/src/main/java/org/apache/hudi/common/model/HoodieCommitMetadata.java

```diff
@@ -119,6 +119,16 @@ public void setCompacted(Boolean compacted) {
     return filePaths;
   }
 
+  public HashMap<String, String> getFileIdAndRelativePathsByPartitionPath(String partitionPath) {
```

Comment:
If we pass `partitionPath` as an argument there's no point in returning a Map -- all files will have the same path, we can just return the file-ids